README.md

```diff
@@ -197,28 +198,51 @@ Per SigMF, the annotations value is an array of annotation segment objects that
 #### 4.3.1 Measurement Types
 The following annotation objects are used within the `scos` SigMF name space associated with `measurement_type`. 
 
-##### SingleFrequencyFFTDetection Object
-Single-frequency FFT detection is a standard software-defined radio measurement. The `SingleFrequencyFFTDetection` object contains the following name/value pairs:  
+##### Detection Object
```

Comment:
What is the purpose of this object? How does it differ from all the other detection objects? Do you mean it is a generic or basic detection algorithm? Maybe call it "BasicDetection Object" or something like that? As "Detection Object", it makes it seem it's a parent object to all the others.

Reply:
Instead of `Detection` v `FFTDetection`, `TimeDomainDetection` v `FrequencyDomainDetection`?